Win11ThemeTest/TextBoxTest.cs

```diff
@@ -0,0 +1,449 @@
+            Assert.IsNotNull(disablexTextBox);
+        }
+
+        //Check if any pre-populated value should be displayed as per requirement.
+        [Test]
+        public void tb1_disabledTextBox()
+        {
+            Assert.IsNotNull(TextWindow);
+            disablexTextBox = TextWindow.FindFirstDescendant(cf => cf.ByAutomationId("tbTxt_disabled")).AsTextBox();    
+            Assert.That(disablexTextBox.Text, Is.EqualTo("TextBox Disabled"));
+        }
+
+        //Check if you cannot edit disabled TextBox.
+        [Test]
+        public void tb1_disabledEditTextBox()
+        {
+            Assert.IsNotNull(TextWindow);
+            disablexTextBox = TextWindow.FindFirstDescendant(cf => cf.ByAutomationId("tbTxt_disabled")).AsTextBox();
+            Assert.That(disablexTextBox.IsEnabled, Is.False);           
+        }
+
+        /* Test Cases for Multi-line Text Box
+         */    
+
+        #endregion
+
+        #region UITests
+
+        [Test]
+        public void tbb1_clickOnTextbox()
+        {
+            Mouse.Click(textBox.GetClickablePoint());
+            Assert.That(textBox.IsEnabled, Is.True);
+        }
+
+        [Test]
+        public void tbb2_mouseHover()
+        {
+            Mouse.MoveTo(textBox.GetClickablePoint());
+            Assert.That(textBox.Text, Is.EqualTo("Hello World!Hello World!Hello World!Hello World!Hello World!Hell World!Hello World!Hello World!Hello World!"));
+        }
+
+        [Test]
+        public void tbb3_fontFamily()
+        {
+            using (var automation = new UIA3Automation())
+            {
+                string expected_FontFamily = "Segoe UI";
+                textBox.Enter("Hello World!");
+                var colorRange = textBox.Patterns.Text.Pattern;
+                var textFont = (string)colorRange.DocumentRange.GetAttributeValue(automation.TextAttributeLibrary.FontName);
+                Assert.That(textFont, Is.EqualTo(expected_FontFamily));
+
+            }
+        }
+
+        [Test]
+        public void tbb4_fontSize()
+        {
+            using (var automation = new UIA3Automation())
+            {
+                // string expected_FontFamily = "Segoe UI";
+                double expected_FontSize = 14;
+                textBox.Enter("Hello World!");
+                var colorRange = textBox.Patterns.Text.Pattern;
+                var textFontSize = colorRange.DocumentRange.GetAttributeValue(automation.TextAttributeLibrary.FontSize);
+                //var backColorInt = colorRange.DocumentRange.GetAttributeValue(TextAttributes.FontSize);
+                Console.WriteLine("textFontSize...........{0}", textFontSize);
+                //Console.WriteLine("backColorInt...........{0}", backColorInt);
+                Assert.That(textFontSize, Is.EqualTo(expected_FontSize));
+            }
+        }
+
+        [Test]
+        public void tbb5_textForegroundColor()
+        {
+            var automation = new UIA3Automation();
```

Comment:
Don't reinitialize automation, we are already initializing this in constructor.

Reply:
removed